samples/node/pub_sub_electron_node/preload_pubsub5.ts

```diff
@@ -0,0 +1,201 @@
+
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+import {mqtt5, iot} from "aws-iot-device-sdk-v2";
+import {ICrtError} from "aws-crt";
+import {once} from "events";
+import { toUtf8 } from '@aws-sdk/util-utf8-browser';
+import * as args from "./settings"
+import { contextBridge } from 'electron';
+
+contextBridge.exposeInMainWorld(
+    'electron',
+    {
+      Mqtt5Connect: () => PubSub5(),
+      Mqtt5Disconnect: () => PubSub5Disconnect(),
+      Mqtt5PublishQoS1: () => PublishTestMessage()
+    }
+  )
+
+function log (msg : string){
+    console.log(msg);
+    let consoleDiv = document.getElementById("console") as HTMLInputElement;
+    let div = document.createElement('div')
+    div.innerHTML = msg;
+    consoleDiv?.appendChild(div);
+}
+
+function creatClientConfig() : mqtt5.Mqtt5ClientConfig {
+    let builder : iot.AwsIotMqtt5ClientConfigBuilder | undefined = undefined;
+
+    if (args.key_file_path && args.cert_file_path) {
+        builder = iot.AwsIotMqtt5ClientConfigBuilder.newDirectMqttBuilderWithMtlsFromPath(
+            args.endpoint,
+            args.cert_file_path,
+            args.key_file_path
+        );
+    } else {
+        let wsOptions : iot.WebsocketSigv4Config | undefined = undefined;
+        if (args.region) {
+            wsOptions = { region: args.region };
+        }
+
+        builder = iot.AwsIotMqtt5ClientConfigBuilder.newWebsocketMqttBuilderWithSigv4Auth(
+            args.endpoint,
+            // the region extraction logic does not work for gamma endpoint formats so pass in region manually
+            wsOptions
```

Comment:
Nitpick: I think you can delete this comment? This sample shouldn't be running against gamma endpoints, or at least I don't think it will/should.